client/src/components/Sub_RatingsReviews/generateStars.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import uuid4 from 'uuid4';
+import Star from './Star.jsx';
+
+export default function generateStars(rating, amtStars, size = 25) {
```

Comment:
I think it is safe to say amtStars can have default too which is value of 5?

Reply:
i agree. will implement this